src/core/src/op/group_normalization.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/group_normalization.hpp"
+
+#include "itt.hpp"
+#include "ngraph/validation_util.hpp"
```

Comment:
Looks like this header is redundant (and all oft the functions there are deprecated).